### PR TITLE
Create README.md

### LINKED ISSUE #6: this repo could use a README
as this image becomes more complicated it becomes difficult to know what options are available to me as a user. it would be nice to have a simple readme file that could contain this information.

### PR DESCRIPTION
**Pull request by willb**
Fixes #6

### CODE REVIEW

**Comment by willb** [comment]
@elmiko ptal

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# base-notebook
+
+This is a container image intended to make it easy to running Jupyter notebooks with Apache Spark on OpenShift.  You can use it as-is (by adding it to a project), or you can use it as the basis for another image.  In the latter case, you'll probably want to add some notebooks, data, and/or additional packages to the derived image.
```
Comment (elmiko) [inline README.md]: s/to running/to run/

**Review by elmiko — CHANGES_REQUESTED** [review]
one minor nit, otherwise lgtm

**Comment by willb** [comment]
Thanks @elmiko!

**Pull request merged**